python/src/trezorlib/cardano.py

```diff
@@ -184,3 +187,29 @@ def _create_change_address_output(output) -> messages.CardanoTxOutputType:
         )
     else:
         raise ValueError("Unknown address type")
+
+
+def create_certificate(certificate) -> messages.CardanoTxCertificateType:
```

Comment:
Add type annotation for certificate if possible.

Reply:
Sadly it's not possible. It's dynamic from JSON.